stl10_input.py

```diff
@@ -130,18 +134,31 @@ def _progress(count, block_size, total_size):
 def save_images(images, labels):
     print("Saving images to disk")
     i = 0
-    for image in images:
-        label = labels[i]
-        directory = './img/' + str(label) + '/'
-        try:
-            os.makedirs(directory, exist_ok=True)
-        except OSError as exc:
-            if exc.errno == errno.EEXIST:
-                pass
-        filename = directory + str(i)
-        print(filename)
-        save_image(image, filename)
-        i = i+1
+    if labels:
```

Comment:
This part has a lot of code duplication - if and else block is practically identical, https://pl.wikipedia.org/wiki/DRY is violated.
If you did that in for loop it would be better, more less like that:
```
        for image in images:
            if labels:
                label = labels[i]
                directory = './img/' + str(label) + '/'
            else:
                directory='./unlabeled_img/'
```

Reply:
Sorry about that, I was rushing writing the code a bit. Would you like me to update the code and submit another pull request? I'm a bit new to the pull request system so I'm not really sure how this process works.

Reply:
No problem :) 

I'm also new to pull request in github, but I think just update the code in your repository and branch, push it and I should see the change in this pull request.